client/src/app/pages/migration-waves/migration-waves.tsx

```diff
@@ -104,11 +104,9 @@ export const MigrationWaves: React.FC = () => {
     migrationWaveModalState !== "create" ? migrationWaveModalState : null;
   const openCreateWaveModal = () => setWaveModalState("create");
 
-  const [exportIssueModalOpen, setExportIssueModalOpen] = React.useState(false);
-
-  const [applicationsToExport, setApplicationsToExport] = React.useState<Ref[]>(
-    []
-  );
+  const [applicationsToExport, setApplicationsToExport] = React.useState<
```

Comment:
No reason to use react state here.

Reply:
Using only that state allowing `null` value simplifies modal handling

Reply:
Ok. Can we do that without using react state? Maybe we can use a variable instead.